### ISSUE TITLE
The "tab" key and others do not work in the bookmarks page

### ISSUE THREAD

**Issue by sgerwk** [luakit/luakit#788]
## Current Behavior
In the bookmark page, the tab, enter, escape keys do not work unless the one of the fields is selected with the mouse.

## Expected Behavior
Tab for navigating the fields, enter to accept and esc to escape, right after pressing 'B'.

## Reproduction Instructions
From any page, press 'B', then for example 'tab', or 'enter'

## Environment
archlinux
luakit 2.1-13-g48c70439
  built with webkit 2.24.1 (installed version: 2.24.2)

**Comment by aidanholm** [comment]
Have you tried entering insert mode? E.g. `B` then `i`, then `tab` etc. It's possible this should be made the default behaviour for `B`.

**Comment by sgerwk** [comment]
Yes, the 'i' key did most of the work. When navigating to the buttons I do not see them highlighted, but maybe this is a different issue.

I 'd vote for the insert mode being the default for the bookmark page: it is entered by keys, and that change would make it immediately reactive to normal keys.
